src/main/resources/application.conf

```diff
@@ -17,6 +17,9 @@ mantis {
   # timeout for shutting down the ActorSystem
   shutdown-timeout = "15.seconds"
 
+  # Whether to run Mantis in test mode (enables test_ RPC endpoints; similar to --test flag in cpp-ethereum)
```

Comment:
Does it change other functionality? If not, could be moved into `rpc` section.

Reply:
It does. If running in testmode we use different Ledger/Consensus implementations.

Reply:
In that case maybe we should document how the functionality differs in test mode? Or is that already documented somewhere else?